boards/arm/lpcxpresso55s36/lpcxpresso55s36-pinctrl.dtsi

```diff
@@ -42,10 +42,94 @@
 	/* Configures pin routing and optionally pin electrical features. */
 	pinmux_sctimer_default: pinmux_sctimer_default {
 		group0 {
-			pinmux = <SCT0_OUT0_PIO1_4>,
-				<SCT0_OUT2_PIO0_15>;
+		  pinmux = <FC2_TXD_SCL_MISO_WS_PIO1_25>;
```

Comment:
Why is the pinmux setting for sctimer changed here? Was the previous definition incorrect?